### PR TITLE
4.view all reviews

### PR DESCRIPTION
**Pull request by belindat98**

### CODE REVIEW

File: src/Components/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    } else {
+        loginMessage = <li className="navbar-item">You are logged in as {currentUser}</li>
+    }
+  return (
+    <section className="navbar">
+      <ul>
+        <li>
+          <Link to="/reviews">Home</Link>
+        </li>
+        {/* <li>
```
Comment (dappernerddesigns) [inline src/Components/Nav.jsx:20]: Make sure to remove commented out code.

File: src/App.jsx [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+import Nav from './Components/Nav'
+import Header from './Components/Header'
+import { useEffect, useState } from 'react'
+import { getAllCategories } from './utils/api'
 
 function App() {
 
+  const [allCategories, setAllCategories] = useState([])
+
+  useEffect(() => {
```
Comment (dappernerddesigns) [inline src/App.jsx:14]: Wouldn't it make more sense to have the navigation component hold this state? Why does it need to be this high?

File: src/Contexts/CurrentUser.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { createContext, useState } from "react";
+
```
Comment (dappernerddesigns) [inline src/Contexts/CurrentUser.jsx:2]: Nice :)

File: src/App.jsx [hunk 4]
```diff
@@ -1,9 +1,29 @@
...
+  }, [])
+
   return (
-    <></>
+    <>
+      <Nav categories={allCategories}/>
+      <Header />
+      <Routes>
+        <Route path="/reviews" element={<ReviewList />} />
+      </Routes>
```
Comment (dappernerddesigns) [inline src/App.jsx:27]: You are missing a route for '/'. This is then giving an error in the browser console that a route is missing when visiting the site.

**Review by dappernerddesigns — APPROVED** [review]
This looks good on mobile and desktop. I'd consider some tweeks for interim screen sizes when you come back to fully style :) things look a little marooned.

I've left some comments I'd like you to consider before you move on, but def on the right track. Love getting your useContext in early, good move.

File: src/Components/Nav.jsx (hunk 1 above)
Comment (belindat98) [reply src/Components/Nav.jsx:20]: This code will be brought back on a later ticket, I removed it for now as the functionality isn't fully there- is this okay?

File: src/App.jsx (hunk 2 above)
Comment (belindat98) [reply src/App.jsx:14]: I'll need access to the allCategories for the filters on a later ticket. Is it okay to leave here or will I need to move up?

File: src/Components/ReviewList.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react"
+import { getReviews } from "../utils/api"
+import ReviewCard from "./ReviewCard";
+
+const ReviewList = () => {
+    const [reviews, setReviews] = useState([])
```
Comment (HannahBurks) [inline src/Components/ReviewList.jsx:6]: We also want to add a loading state that will display a loading message if awaiting data from api still. 

Really nice styling and responsive design though!

Once amended, please send another pr :)

**Review by HannahBurks — CHANGES_REQUESTED** [review]

**Review by HannahBurks — APPROVED** [review]
Looks great Belinda 👏🏼 Merge away! 😄
